src/plugins/workspace/server/routes/index.ts

```diff
@@ -42,9 +42,33 @@ const settingsSchema = schema.object({
   dataSources: schema.maybe(dataSourceIds),
 });
 
+const featuresSchema = schema.arrayOf(schema.string(), {
+  minSize: 1,
+  validate: (featureConfigs) => {
+    const validateUseCaseConfigs = [
+      'all',
+      'observability',
+      'security-analytics',
+      'analytics',
+      'search',
```

Comment:
Can we get the ids from default_nav_group.ts?

Reply:
Sure, I got an `error  Unexpected path "../../../../core/utils" imported in restricted zone. Plugins may only import from src/core/server and src/core/public  @osd/eslint/no-restricted-paths` when commit changes . But after I change to `import { DEFAULT_NAV_GROUPS } from '../../../../core/public';`, the commit goes well. Will update this.